commands/addons/attach.js

```diff
@@ -43,7 +43,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'addons',
   command: 'attach',
-  description: 'attach add-on resource to a new app',
+  description: 'attach an existing add-on resource to an app',
```

Comment:
If we're thinking about things, do we like the word "Resource" or should we think about using "Installation" or something like that? 

Just a question. I don't have a strong opinion here.